.github/CONTRIBUTING.md

```diff
@@ -86,234 +103,258 @@ git remote add upstream git@github.com:carbon-design-system/carbon.git
+-->
 
-After your changes are committed, run:
+See
+[How to write the perfect pull request](https://github.com/blog/1943-how-to-write-the-perfect-pull-request)
+for tips on writing a good PR description.
 
-```sh
-git push -u origin { YOUR_BRANCH_NAME }
-```
-
-In your browser, navigate to
-[carbon-design-system/carbon-](https://github.com/carbon-design-system/carbon)
-and click the button that reads `Compare & pull request`
-
-Write a title and description then click `Create pull request`
+#### Update a pull request
 
-- [How to write the perfect pull request](https://github.com/blog/1943-how-to-write-the-perfect-pull-request)
+<!--
+TODO: Doesn't seem like it should need to be explained how to update a pull
+request. This process should be documented in the GitHub docs.
+-->
 
-### 9. Updating a pull request
+Stay up to date with activity in your PR. Maintainers will review your work,
+make comments, ask questions, and suggest changes.
 
-Stay up to date with the activity in your pull request. Maintainers from the
-Design System team will be reviewing your work and making comments, asking
-questions and suggesting changes to be made before they merge your code.
+When you need to make a change, use the same method detailed in the previous
+section to make additional commits on your branch. Once you push those changes
+to your fork on GitHub, the PR will automatically update with your new commits.
 
-When you need to make a change, use the same method detailed above except you no
-longer need to run `git push -u origin { YOUR_BRANCH_NAME }` just `git push`.
-
-Once all revisions to your pull request are complete, someone from Design
-Systems will squash and merge your commits for you.
+Once the changes are complete and your PR is approved, a maintainer will merge
+your changes.
 
 ## FAQ
 
 ### Who can contribute?
 
-Anyone! We mean it. The one and only requirement is you'll need a
-[public GitHub account](https://github.com/join), as all our assets live on
-GitHub.
+Anyone! The only requirement is a public GitHub account as all our assets live
+on GitHub. Sign up on https://github.com.
 
 - **Development:** If coding is your thing, you can help us by contributing bug
-  fixes or community components. Checkout our
-  [Developer Handbook](../docs/developer-handbook.md) to get your dev
-  environment set up, read up on our best practices and more.
-- **Design:** Design contributions can vary from visual assets, UX interactions,
-  motion design, Sketch kit bug fixes and more.
-- **Content:** Our documentation is just as important as our design and code
-  assets. Whether it's updating our current docs, or adding new
-  [patterns](./patterns), anyone can contribute to our
+  fixes or community components. Check out our
+  [Developer Handbook](../docs/developer-handbook.md) for setup and best
+  practices.
+  <!--
+  TODO: Are non-maintainers allowed to contribute design guidance? That seems to
+  be what this point is implying.
```

Comment:
Yep, we get contributions from designers that participate in internal working groups and such.